test/routes/bespokeApplyOffline.route.test.js

```diff
@@ -65,7 +71,7 @@ Object.entries(routesToTest).forEach(([route, { path: routePath, itemType, pageH
     lab.beforeEach(() => {
       mocks = new Mocks()
 
-      Object.assign(mocks.wasteActivities, { activities: 'item-1,item-2,item-3,item-4' })
+      mocks.taskDeterminants.wasteActivities = ['item-1', 'item-2', 'item-3', 'item-4']
```

Comment:
This seems to now be testing something different, possibly the correct thing but the state of the `mocks` object is not clear at all ☹️ 

Not sure if this is just a comment or an actual concern, sorry. 😢

Reply:
I've added comments to hopefully clarify what's happening with the activities tests